volatility3/framework/plugins/linux/threads.py

```diff
@@ -0,0 +1,67 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
This is very trivial, but it would be nice if the license is changed to 2022.
```suggestion
# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
```